microprofile/metrics/metrics-se/src/test/java/io/helidon/metrics/HelidonHistogramTest.java

```diff
@@ -84,7 +85,7 @@
     private static HelidonHistogram delegatingHistoLong;
 
     @BeforeAll
-    static void initClass() {
+    static void initClass() throws Exception {
```

Comment:
Why is the "throws Exception" added? As far as I can see there is no exception thrown by any of the statements.

Reply:
That's correct, was used for some testing but it is not needed anymore. I will remove clause.